content/en/Integrations/IntegrationBuilder/configure-gitlab-integration.md

```diff
@@ -8,17 +8,22 @@ weight: 40
 
 To configure GitLab to create issues for new findings, you need three general steps:
```

Comment:
This is still mentioning three steps, although I think we can leave it like that if we remove the Cobalt connection part.